broker-plugin/plugin/src/main/resources/shared/login.config

```diff
@@ -0,0 +1,26 @@
+PropertiesLogin {
+   org.apache.activemq.artemis.spi.core.security.jaas.PropertiesLoginModule required
+       debug=false
+       org.apache.activemq.jaas.properties.user="artemis-users.properties"
+       org.apache.activemq.jaas.properties.role="artemis-roles.properties";
+};
+
+CertLogin {
+   org.apache.activemq.artemis.spi.core.security.jaas.TextFileCertificateLoginModule required
+       debug=true
+       org.apache.activemq.jaas.textfiledn.user="cert-users.properties"
+       org.apache.activemq.jaas.textfiledn.role="cert-roles.properties";
+};
+
+activemq {
```

Comment:
redundant block?

Reply:
This is not need I think, I copied it from https://github.com/k-wall/artemis-dual-login-module/blob/master/etc/login.config :)